ex1B.py

```diff
@@ -0,0 +1,15 @@
+def numdivision(numlist , num ):
+    numinlist  = len(numlist)
+    counter = 0
+    for numbers in range(numinlist ) :
+        if numlist[counter] % num == 0 :
+            for extranum in range(numinlist - numbers - 1):
+                numlist[counter + extranum] = numlist[counter + extranum + 1]
+            counter = counter-1
+            numlist.pop()
+        counter = counter + 1
+    return numlist
+
```

Comment:
where is the second way - with list comprehension?

Reply:
i did only section b there is also section c